From 8d2aefecce7cca696f44d0792e196ecdd22072e4 Mon Sep 17 00:00:00 2001
From: Ralf Haferkamp <rhafer@suse.com>
Date: Fri, 3 Jul 2020 11:35:52 +0200
Subject: [PATCH] Fix config_from_options() to correctly enable ipv6

Signed-off-by: Ralf Haferkamp <rhafer@suse.com>
---
 main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/main.c
+++ b/main.c
@@ -680,7 +680,7 @@
     if (rc < 0) {
         goto finish;
     }
-    cfg->enable_ipv6 = cfg->enable_ipv6;
+    cfg->enable_ipv6 = opt->enable_ipv6;
     cfg->disable_host_loopback = opt->disable_host_loopback;
     cfg->enable_sandbox = opt->enable_sandbox;
     cfg->enable_seccomp = opt->enable_seccomp;
